README.md

```diff
@@ -90,12 +90,12 @@ Rel(web_app, twitter, "Gets tweets from", "HTTPS")
 
 ## Snippets for Visual Studio Code
 
-Because the PlantUML support inside of Visual Studio Code is excellend with the [PlantUML extension](https://marketplace.visualstudio.com/items?itemName=jebbs.plantuml), you can also find VS Code snippets for C4-PlantUML at [.vscode/snippets/diagram.json](.vscode/snippets/diagram.json).
+Because the PlantUML support inside of Visual Studio Code is excellent with the [PlantUML extension](https://marketplace.visualstudio.com/items?itemName=jebbs.plantuml), you can also find VS Code snippets for C4-PlantUML at [.vscode/C4.code-snippets](.vscode/C4.code-snippets).
```

Comment:
.code-snippets files have a slightly different schema than snippet JSON files since they need to specify which languages the snippets apply to. Simply renaming `.vscode/snippets/diagram.json` to `.vscode/C4.code-snippets` won't work according to VS Code documentation, or will lead to the snippets appearing everywhere, not just in PlantUML diagrams.

Your PR should add the `scope` property to each snippet as described in [the docs](https://code.visualstudio.com/docs/editor/userdefinedsnippets "Creating your own snippets"), or better yet roll back the project-level snippet stuff and give it its own PR since it is disjoint from the purpose of this PR.

Reply:
Ok. Rolling it back. Forgot to switch branch before pushing